VnE/Source/MainWindow.py

```
@@ -235,9 +245,40 @@ def select(self, pos):
                 else:
                     self.selection_model.select(index, QItemSelectionModel.Select)
             self.update()
-
         return points
 
+    def screenshot(self, filename, res=None):
+        if not res:
+            res = glGetIntegerv(GL_VIEWPORT)[2:]
```

Ruff (0.15.6)

[error] `glGetIntegerv` may be undefined, or defined from star imports

(F405)